packages/pgocaml/pgocaml.3.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "dev@ocsigen.org"
+synopsis: "Interface to PostgreSQL databases"
+description: "PG'OCaml provides an interface to PostgreSQL databases for OCaml applications. It uses Camlp4 to extend the OCaml syntax, enabling one to directly embed SQL statements inside the OCaml code. Moreover, it uses the describe feature of PostgreSQL to obtain type information about the database. This allows PG'OCaml to check at compile-time if the program is indeed consistent with the database structure. This type-safe database access is the primary advantage that PG'OCaml has over other PostgreSQL bindings for OCaml."
+authors: ["Richard W.M. Jones <rich@annexia.org>"]
+homepage: "http://pgocaml.forge.ocamlcore.org/"
+bug-reports: "https://github.com/darioteixeira/pgocaml/issues"
+dev-repo: "git+https://github.com/darioteixeira/pgocaml.git"
+license: "LGPL-2.0 with OCaml linking exception"
+build: [
+  ["./configure" "--%{camlp4:enable}%-p4" "--prefix" prefix "--docdir" "%{doc}%/pgocaml"]
+  [make]
+  [make "doc"]
+]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "pgocaml"]]
+depends: [
+  "ocaml"
+  "base-bytes"
+  "calendar"
+  "csv"
+  "hex"
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
+  "ounit" {with-test}
+  "re"
+	"camlp4"
```

Comment:
Why is this no longer a depopt?

Reply:
I can't seem to get the configuration right. For now I'm trying to get it through Travis CI as a dependency, then get back to the depext issue.

Reply:
@jrochel the camlp4 failure on Travis wasn't your fault. It was due to a too old opam version being incorrecly used by the CI. Can you please move calp4 back in the depext and rebase on the current opam-repository? The problem should disappear now.

Reply:
Thanks. Camlp4 is again a depext. The sole remaining problem is:
> Fatal error: cannot load shared library dllunix

I'm guessing that this is not directly related to pgocaml...

Reply:
Mmh, I thought that should have been part of the solved issues. @dra27 can you also have a look at the Travis failure?

Reply:
On it!

Reply:
#13750!

Reply:
Thanks! Do we need to wait for #13750 to be merged? Otherwise I'd say that this PR should be fine...

Reply:
On the basis that I thought the previous fix would deal with it (the message is the same, but on different build configurations), I’d prefer to see this one pass CI first, but I don’t think any further action will be needed on this PR, beyond perhaps a rebase and squash after #13750 in order to kick the CI to run on a fresh merge commit

Reply:
Alright.